jerry-core/jerry-port.h

```diff
@@ -71,6 +72,34 @@ typedef enum
  */
 void jerry_port_fatal (jerry_fatal_code_t code);
 
+/*
+ * Date Port API
+ */
+
+/**
+ * Jerry time zone structure
+ */
+typedef struct
+{
+  int offset;                /**< minutes from west */
+  int daylight_saving_time;  /**< daylight saving time (1 - DST applies, 0 - not on DST) */
+} jerry_time_zone_t;
+
+/**
+ * Get timezone and daylight saving data
+ *
+ * @return true  - if success
+ *         false - otherwise
+ */
+bool jerry_port_get_time_zone (jerry_time_zone_t *tz_p);
+
+/**
+ * Get system time
+ *
+ * @return milliseconds since Unix epoch
+ */
+unsigned long int jerry_port_get_current_time (void);
```

Comment:
We have to use milliseconds everywhere, so it is the best, so we don't have to use multiplications and divisions. I think ms is good.